elyra/pipeline/processor_kfp.py

```diff
@@ -67,8 +72,22 @@ def process(self, pipeline):
             self.log.info("Kubeflow Pipeline successfully compiled.")
             self.log.debug("Kubeflow Pipeline was created in %s", pipeline_path)
 
-            # Upload the compiled pipeline and create an experiment and run
-            client = kfp.Client(host=api_endpoint)
+            # Upload the compiled pipeline, create an experiment and run
+
+            session_cookie = None
+
+            # TODO: double check if empty string is a valid pwd
+            if api_endpoint_username and api_endpoint_password:
+                endpoint = api_endpoint.replace('/pipeline', '')
+                session_cookie = self._get_user_auth_session_cookie(endpoint,
+                                                                    api_endpoint_username,
+                                                                    api_endpoint_password)
+
+            client = kfp.Client(
```

Comment:
Would you mind joining these lines into a single line for the statement?  I think it reads easier and, although I like whitespace, this seems to introduce a lot of it.